src/crimson/os/seastore/transaction_manager.h

```diff
@@ -961,7 +974,9 @@ class TransactionManager : public ExtentCallbackInterface {
 	original_laddr,
 	std::move(original_bptr));
       fut = lba_manager->alloc_extent(
-	t, remap_laddr, remap_length, remap_paddr, *ext);
+	t, remap_laddr, remap_length, remap_paddr,
+	(original_bptr.has_value() ? ext->get_crc32c() : 0),
```

Comment:
Add a TODO comment that the original_bptr must be present if crc is enabled.

Reply:
ok, by the way, I've implemented the `max extent size` PR (https://github.com/ceph/ceph/pull/55682) which can make sure that the original_bptr here always have a value when crc is enabled. Maybe I should merge commits from https://github.com/ceph/ceph/pull/55682 here? What do you think?

Reply:
It's up-to-you, but TODO comment would still be perferred if this commit is arranged before the follow-up fixes.